tests/functional/conftest.py

```diff
@@ -39,7 +39,7 @@ def backup_stats_data():
     random_result_code = random.choice(range(4))
     return {
         "duration": random.random() * 100,
-        "status_ok": random_result_code != 1,
+        "status_ok": float(random_result_code == 0),
```

Comment:
Why we are using float hear? `1.0` and `0.0` are needed?

Reply:
+1

Reply:
`boo`l / `float` / `int` are both okay, `prometheus_client` will internally convert it back to `float` since it is a gauge metric. However, to be consistent with the expected input of a gauge metric, I explicitly convert it to`float`.

Reply:
I think it's better to simulate the output of collector.
In the function `test_backup_stats_collector`, the regular expression is used to extract "integer" , isn't?

"status_ok": r"juju_backup_all_command_ok_info{.*} [\d].*",

So, I think it's unneccessary to change it to float. Does that make sense?

Reply:
Done